README.md

```diff
@@ -193,6 +193,15 @@ for doing very specific things.
 | load the analyzer tool on a stats file                                                                      | `yarn analyze`                                                                                                                                                                                                               |
 | add a new React app                                                                                         | `yarn new:app` (make sure you have [`vagov-content`](https://github.com/department-of-veterans-affairs/vagov-content/) sibling to `vets-website`)                                                                            |
 
+## Debugging React with VS Code
+
+1. Install [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) extension.
```

Comment:
What does this do? Does it let you step through things in vscode?

Reply:
yup exactly, you can set breakpoints, step through code, etc. in vscode itself instead of chrome dev tools